src/components/Dapp.res

```diff
@@ -554,19 +554,16 @@ module DefaultLook = {
 module DefaultLeftPanel = {
   @react.component
   let make = () => {
-    let translationModeContext = ReactTranslate.useTranslationModeContext()
-    let translation = ReactTranslate.useTranslate(. translationModeContext.translationMode)
     <React.Fragment>
       <h1 className=Styles.heading>
-        <span className=Styles.colorBlue> {translation(. "bluetext")->restr} </span>
+        <span className=Styles.colorBlue> {"Always"->restr} </span>
         <br />
-        {translation(. "ethereum")->restr}
+        {"raising funds for"->restr}
         <br />
         <span className=Styles.colorGreen> {"conservation"->restr} </span>
-        {(" " ++ translation(. "tokens"))->restr}
       </h1>
       <hr />
-      <h3 className=Styles.subHeading> {translation(. "subHeading")->restr} </h3>
+      <h3 className=Styles.subHeading> {"Adopt an animal to start giving today."->restr} </h3>
```

Comment:
The muggle toggle wasn't obvious or clear here. My friend couldn't work out what the hell the point was of it, and I think most users are more confused by it than relieved. We should add it tastefully to various interfaces and explanations. But shouldn't be on home page imo.

Reply:
Sounds like you got good feedback on this so happy with the simplification